### DISCUSSION TITLE
is it possible to increase drei Html element only on height?

### DISCUSSION THREAD

**Discussion by Achi00** [pmndrs/drei#861] [Q&A] [answered]
I have put drei html on gltf models screen so i need to fill spaces to make it look good and only scale i need is height

here is image
![image](https://user-images.githubusercontent.com/101459158/163431856-546b6b4a-6517-4169-b5c7-159b2091bd4b.png)
thanks.

**Comment by mwmwmw** [comment, marked as answer]
You can style the `Html` component directly using the style property. You can also assign it a class and use CSS (probably the cleanest way)

```javascript
<Html
  transform
  center
  style={{
    width: "100px",
    height: "75px", 
    backgroundColor: "#454545",
    display: "grid",
    placeContent: "center"
  }}
>
  <h1 style={{ color: "white", fontFamily: "sans-serif" }}>Hello</h1>
</Html>
```

https://codesandbox.io/s/drei-html-styling-example-pkb811?file=/src/index.js

You can review the other properties available to `Html` here. 
https://github.com/pmndrs/drei#html
